Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -302,13 +314,14 @@ public class ContainerStackView: UIView {
         }
     }
 
-    public func showSubview(_ subview: UIView, animated: Bool = true) {
+    /// Shows the arranged subview by setting the width, height and spacing constraints to the original value before being hidden.
+    private func showArrangedSubview(_ subview: UIView) {
         guard subviews.contains(subview) else { return }
         guard subview.alpha == 0 else { return }
 
         updateConstraintsIfNeeded()
 
-        Animate(isAnimated: animated) {
+        Animate(isAnimated: true) {
```

Comment:
Do you think we need the implicit animations inside the container view? 🤔What if by default the hide/show works without any animation but if we need one we can be explicit:
```swift
Animate { subview.isHidden = true }
```

Reply:
Ah yes, you are totally right! Will change this 👍

Reply:
✅  Done!